sr/robot3/kch.py

```diff
@@ -364,49 +297,11 @@ def __init__(self, led_num: int, server: LedServer) -> None:
         self._led_num = led_num
         self._server = server
 
-    @property
-    def r(self) -> bool:
-        """Get the state of the Red LED segment."""
-        return self._server.get_leds(self._led_num)[0]
-
-    @r.setter
-    def r(self, value: bool) -> None:
-        """Set the state of the Red LED segment."""
-        # Fetch the current state of the LED so we can update only the red value
-        current = self._server.get_leds(self._led_num)
-        self._server.set_leds(self._led_num, (value, current[1], current[2]))
-
-    @property
-    def g(self) -> bool:
-        """Get the state of the Green LED segment."""
-        return self._server.get_leds(self._led_num)[1]
-
-    @g.setter
-    def g(self, value: bool) -> None:
-        """Set the state of the Green LED segment."""
-        # Fetch the current state of the LED so we can update only the green value
-        current = self._server.get_leds(self._led_num)
-        self._server.set_leds(self._led_num, (current[0], value, current[2]))
-
-    @property
-    def b(self) -> bool:
-        """Get the state of the Blue LED segment."""
-        return self._server.get_leds(self._led_num)[2]
-
-    @b.setter
-    def b(self, value: bool) -> None:
-        """Set the state of the Blue LED segment."""
-        # Fetch the current state of the LED so we can update only the blue value
-        current = self._server.get_leds(self._led_num)
-        self._server.set_leds(self._led_num, (current[0], current[1], value))
-
-    @property
-    def colour(self) -> tuple[bool, bool, bool]:
+    def get_colour(self) -> tuple[bool, bool, bool]:
         """Get the colour of the user LED."""
         return self._server.get_leds(self._led_num)
 
-    @colour.setter
-    def colour(self, value: tuple[bool, bool, bool]) -> None:
+    def ste_colour(self, value: tuple[bool, bool, bool]) -> None:
```

Comment:
```suggestion
    def set_colour(self, value: tuple[bool, bool, bool]) -> None:
```

Interesting that tests didn't pick up on this.